scenariolib/event_pageview.go

```diff
@@ -1,51 +1,82 @@
 package scenariolib
 
-import "errors"
+import (
+	"errors"
+	"math"
+	"math/rand"
+)
 
 // ============== VIEW EVENT ======================
-// ==================================================
+// =================================================
 
-// ViewEvent a struct representing a search, is defined by a query to execute
+// ViewEvent a struct representing a view, it is defined by a clickRank, an
+// offset, a probability to click and the contentType
 type ViewEvent struct {
-	pageURI         string
-	pageReferrer    string
-	pageTitle       string
-	contentIdKey    string
-	contentIdValue  string
-	contentType     string
+	clickRank   int
```

Comment:
I did not change what is sent to UA, but mostly what is required for the scenario. Before the scenario required that you specify URL, title, etc.. but now this information is obtained from the search results (like the Click event), so no longer required. But we now need the same offset, docNo and probability.